src/core/components/Paginate/Paginate.scss

```diff
@@ -1,13 +1,43 @@
-.paginator {
+@import '~core/css/inc/mixins';
+
+.Paginator::after {
+  clear: both;
+  content: '';
+  display: block;
+}
+
+.Paginator-page-number {
+  font-weight: bold;
+  margin: 0 0 5px;
+  text-align: center;
+}
+
+.Paginator-links {
   display: flex;
   justify-content: space-between;
-  margin: 1em;
-  padding: 0;
 }
 
-.paginator--item {
+.Paginator .Paginator-item {
+  @include button($background: #fff, $color: #000);
+
   display: inline-block;
+  font-size: $font-size-s;
+  font-variant: small-caps;
+  font-weight: normal;
+  line-height: 2;
   margin: 0 0.5em;
-  list-style-type: none;
+  min-width: 40%;
   text-align: center;
 }
+
+.Paginator-previous {
```

Comment:
Ah, yeah, when I looked at it in Arabic the Next was on the left instead of the right with the styles as-is, which I assumed was correct.

I will file an issue to keep it on our radar.